requirements.txt

```diff
@@ -73,12 +77,15 @@ pyasn1==0.4.2
 pyasn1-modules==0.2.1
 pycparser==2.18
 python-dateutil==2.6.1
+python-magic==0.4.15
 pytz==2017.3
 pyvips==2.0.4
 requests==2.18.4
 rsa==3.4.2
+s3cmd==2.0.2
```

Comment:
This may not load.  I've been having to manually download version 2.0.2, which lets you set the endpoint to the digitalocean server rather than the AWS default.